docs/index.rst

```diff
@@ -38,7 +38,7 @@ speed right away.
 * Like learning by example? Visit the official :ref:`html_tutorials` as well as
   the community-contributed :ref:`Pyramid Tutorials
   <tutorials:pyramid-tutorials>` and :ref:`Pyramid Community Cookbook
-  <cookbook:pyramid-cookbook>`.
+  <cookbook:pyramid_cookbook>`.
```

Comment:
Please revert. See failing GitHub Action https://github.com/Pylons/pyramid/actions/runs/4924847110/jobs/8799580436?pr=3727#step:5:574 and actual target https://github.com/Pylons/pyramid_cookbook/blame/master/docs/index.rst#L1